### DISCUSSION TITLE
Formula from tap not updated

### DISCUSSION THREAD

**Discussion by mthmulders** [Homebrew/discussions#120] [Everyday usage] [answered]
Hi all,

I've created [a tap](https://github.com/mthmulders/homebrew-maven-snapshot/) with [one formula](https://github.com/mthmulders/homebrew-maven-snapshot/blob/master/Formula/maven-snapshot.rb). That formula is auto-updated using [GitHub actions](https://github.com/mthmulders/homebrew-maven-snapshot/actions?query=workflow%3A%22update+formula%22).

I've installed that tap on my Mac, with `brew tap mthmulders/maven-snapshot` but no matter how often the formula is updated, I don't get a new version when I do `brew upgrade`. What am I doing wrong here?

Thanks in advance!

**Comment by jonchang** [comment, marked as answer]
I looked at the commit history of that formula and it’s not upgrading because the version number is staying the same. If you want the same version number you’ll need to increment the `revision`

**Reply by mthmulders** [reply]
Thanks, that did the trick!
